src/index.ts

```diff
@@ -83,13 +83,13 @@ async function tailwindCSSModule (moduleOptions) {
     tailwindConfig._hash = String(Date.now())
 
     if (options.jit === true) {
-      logger.warn('`tailwindcss.jit` option had been deprecated in favour of tailwind config `mode: \'jit\'`')
-      tailwindConfig.mode = 'jit'
+      logger.warn('`tailwindcss.jit` option is unnecessary as it\'s now the default')
     }
 
-    if (tailwindConfig.mode === 'jit') {
-      logger.info('Tailwind JIT activated')
+    if (options.mode === 'jit') {
+      logger.warn('`tailwindcss.mode: \'jit\'` option is unnecessary as it\'s now the default')
```

Comment:
Would you want to warn for using the "mode" anyway?
As iirc there's no other modes now.

Reply:
If I understand you correctly in that you propose for it to show a warning whenever a mode is set, that would mean any upcoming mode - if there will be any - will throw a warning unless accounted for again in this repo's code. I'd stay at warning only for the default mode.